shoop/front/template_helpers/general.py

```diff
@@ -16,41 +16,50 @@
 from shoop.utils.translation import cache_translations_for_tree
 
 
-def _get_list_products(request):
-    return Product.objects.list_visible(
+def _get_list_products(context, n_products, ordering=None, filter_dict=None, orderable_only=True):
```

Comment:
Docstring missing.

Why does this require `context` just to extract `request` out of it? :)